--- fssm-0.2.10/spec/monitor_spec.rb.orig	2010-09-08 23:45:00.000000000 +0200
+++ fssm-0.2.10/spec/monitor_spec.rb	2026-03-20 15:24:14.000000000 +0100
@@ -60,7 +60,7 @@
       it "should call create callback upon file creation" do
         run_monitor(1) do
           file = @tmp_dir + "/newfile.rb"
-          File.exists?(file).should be_false
+          File.exist?(file).should be false
           FileUtils.touch file
         end
         @handler_results[:create].should == [[@tmp_dir, 'newfile.rb']]
